modules/mri_violations/jsx/Not_Resolved_columnFormatter.js

```diff
@@ -0,0 +1,71 @@
+/* exported formatColumn */
+
+/**
+ * Modify behaviour of specified column cells in the Data Table component
+ * @param {string} column - column name
+ * @param {string} cell - cell content
+ * @param {arrray} rowData - array of cell contents for a specific row
+ * @param {arrray} rowHeaders - array of table headers (column names)
+ * @return {*} a formated table cell for a given column
+ */
+function formatColumn(column, cell, rowData, rowHeaders) {
+  if (loris.hiddenHeaders.indexOf(column) > -1) {
+    return null;
+  }
+  // Create the mapping between rowHeaders and rowData in a row object.
+  var row = {};
+  var hashName;
+  var patientname;
+  var uid;
+  var url;
+  rowHeaders.forEach(function(header, index) {
+    row[header] = rowData[index];
+  }, this);
+
+  if (column === "Problem" && row.Problem === "Protocol Violation") {
+    patientname = row.PatientName;
+    uid = row.SeriesUID;
+    url = loris.BaseURL +
+        "/mri_violations/?submenu=mri_protocol_check_violations&PatientName=" +
+             patientname + "&SeriesUID=" + uid;
+    return <td>
+            <a href= {url}
+            className="mri_violations"
+            id="mri_protocol_check_violations"
+            data-patientname= {patientname}
+            data-seriesuid={uid}
+            >Protocol Violation</a>
+           </td>;
+  }
+  if (column === "Problem" && row.Problem === "Could not identify scan type") {
```

Comment:
If the `if` conditions don't clash, you can probably use a single `columnFormatter.js` file for all tabs (i.e resolved, not resolved, etc).

Otherwise, at least name them consistently (e.g `columnFormatterResolved.js`, `columnFormatterUnresolved.js`, etc)